docroot/themes/custom/mappy/mappy.libraries.yml

```diff
@@ -0,0 +1,14 @@
+global-styling:
+  css:
+    theme:
+      css/styles.css: {}
+
+leaflet:
+  css:
+    theme:
+      css/leaflet.css: {}
```

Comment:
This file actually needs to be in the sass directory named leaflet.scss and then it will get compiled into /css.. the file itself didn't get committed this time b/c the css directory is in .gitignore